tools/snort2lua/helpers/parse_cmd_line.cc

```diff
@@ -280,6 +280,7 @@ static void set_ips_pattern(const char* /*key*/, const char* val)
 static void print_version(const char* /*key*/, const char* /*val*/)
```

Comment:
This needs to become noreturn. See eg print_binder_order.

Reply:
Should be fine now.